nextstrain/cli/util.py

```diff
@@ -129,49 +130,56 @@ def check_for_new_version():
         pkgreq = shquote(f"nextstrain-cli=={newer_version}")
 
         print("A new version of nextstrain-cli, %s, is available!  You're running %s." % (newer_version, __version__))
+        print("See below for upgrade instructions.")
         print()
         print("See what's new in the changelog:")
         print()
         print(f"    https://github.com/nextstrain/cli/blob/{newer_version}/CHANGES.md#readme")
         print()
```

Comment:
Oh, hmm. I wasn't expecting the split of "new version available" from "here's how to upgrade". I'd think to keep the two together, i.e. all at the bottom. In particular, the changelog link is gonna get lost which really should inform if someone wants to upgrade just yet or not. Was there a reason to keep the notice at the top?